Dokument/Dateien/Magie_Essenz.tex

```diff
@@ -1,103 +1,876 @@
+Daher behalten wir sie uns als Option drin, insbesondere für eine spätere Nutzung im entsprechenden Zeitalter.
+
+Sollte jemand sich mit der Thematik des Rundfunks besser auskennen und doch dafür plädieren (und es damit als eine Art Telepathie einsetzen wollen), dann bitte zur Diskussion melden.
+
+\subsubsection{Verbreitung}
+Keine Verbreitung unter den Lebewesen.
+
+\subsubsection{Bedeutung im Spiel}
+Derzeit keine Bedeutung im Spiel.
+\\ \\
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+\subsection{Schub}\label{magie:schub}
+Schub steht für die mechanische Energie, die auf etwas appliziert werden kann.
+Diese Magieart entspricht damit dem Bild einer erweiterten Hand, die diese Kraft ausübt.
+
+
+Zur Unterscheidung von \nameref{magie:druck} kann man sich hier vor Augen halten, dass der Druck auf alles an der entsprechenden Stelle gilt, da er außerhalb angesetzt wird (das kann auch Auswirkungen auf das Objekt selbst haben!). 
+Die mechanische Kraft wirkt jedoch nur auf ein bestimmtes Objekt.
+Damit sind bei ähnlichen Anwendungen präzisere Ausführungen möglich.
+Zudem kann je nach Umstand der Druck an seine Grenzen kommen, wo die mechanische Kraft noch keine Grenzen hat.
+
+Weiterhin unterscheidet sie sich vom Druck darin, dass es immer um eine Bewegung geht.
+Kann mittels Druck etwas zerquetscht werden, so ist dies mit Schub nicht möglich: dazu müsste man andere Dinge nehmen, die den Schub Richtung des zu Zerquetschendes erhalten.
+
+\subsubsection{Verbreitung}
+Eine normale, weit verbreitete Fähigkeit.
+
+\subsubsection{Bedeutung im Spiel}
+Eine normale Fähigkeit unter der Bevölkerung.
+
+\subsubsection{Intuitive Nutzung}
+Ideen pls.
+
+\subsubsection{Erlernbare Steigerungen}
+\begin{outline}
+	\1 Levitation
+	\1 Telekinese
+	\1 ...
+\end{outline}
+
+\subsubsection{Meisterliche Beherrschung} 
+\begin{outline}
+	\1 Herumschieben von anderen Personen.
+	\1 ...
+\end{outline}
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
+
+
+
+\subsection{Temperatur}\label{magie:temperatur}
+Diese Eigenschaft erlaubt die Kontrolle über die thermische Energie der Umgebung, sowohl das Erhitzen als auch das Abkühlen.
+
+\subsubsection{Verbreitung}
+Diese Manipulation der thermischen Energie ist uralt und daher weit verbreitet unter allen Arten von Lebewesen; Somit auch unter den homininen Arten. 
+Sie ist gut bekannt.
+
+\subsubsection{Bedeutung im Spiel}
+Diese Magiekontrolle beherrscht der \npref{sec:mc-soldat}. Es ist ansonsten eine der am weitesten verbreiteten Arten unter der Bevölkerung.
+
+\subsubsection{Intuitive Nutzung}
+Diese Lebewesen haben weniger Temperaturprobleme, da sie den Temperaturaustausch mit der Umgebung reduzieren können.
+Einige Tiere nutzen dieses Konzept auch zur Jagd, sowohl an Land als auch im Wasser, wo sie die Luft oder das Wasser stark erhitzen oder extrem abkühlen und damit für Verbrennungen oder Erfrierungen sorgen.
```

Comment:
ein bisschen sehr extrem für intuitive nutzung, würde ich sagen. das könnte eher bei wechselwarmen beutetieren einen nutzen haben oder um insekten anzulocken/wegzuscheuchen

Reply:
das stimmt, tatsächlich habe ich bei einigen intuitiven Beschreibungen den Tiernutzen auch in seinen extremeren Varianten aufgeführt. Das liegt daran, dass die meisten so beschriebenen Tierarten idR das trotzdem intuitiv nutzen. Nur haben sie zB einen viel größeren Manapool als der Mensch und können damit krasseres bewirken...